marklogic-data-hub-central/ui/src/components/explore/entity-type-display-settings-modal/entity-type-display-settings-modal.tsx

```diff
@@ -73,6 +111,35 @@ const EntityTypeDisplaySettingsModal: React.FC<Props> = ({isVisible, hubCentralC
     return headerLabel;
   };
 
+  const handleSave = () => {
+    try {
+      let updatedPayload = {...hubCentralConfig};
+      for (const key in exploreSettingsData) {
```

Comment:
See if we can avoid this iteration as well, like i mentioned in another comment above.